### DISCUSSION TITLE
运行相关问题

### DISCUSSION THREAD

**Discussion by Indifference061** [mikumifa/biliTickerBuy#77] [提问专区] [answered]
请问填入了id后点击获取票信息按钮没有反应，也没有填入购票人信息的表单是怎么回事呢？
![image](https://github.com/mikumifa/biliTickerBuy/assets/87850383/45445394-0b44-4d2a-a13f-3794833d5ec3)

**Comment by mikumifa** [comment]
确定登陆成功了吗？会弹出浏览器然后登陆

**Reply by Indifference061** [reply]
已经登录过了

**Reply by Indifference061** [reply, marked as answer]
重新了下载一次，现在可以了！
